code/wdb/datastripe.q

```diff
@@ -93,7 +106,7 @@ savetablesoverperiod:{[dir;tablename;nextp;lasttime]
     partitionlist: ?[tablename;();();(distinct;keycol)];
 
     /- enumerate and then split by keycol
-    symdir:` sv dir,.proc.procname;
+    symdir:.wdb.hdbdir;
     enumkeycol: .Q.en[symdir;?[tablename;enlist (<;`time;nextp);0b;()]];
```

Comment:
```suggestion
    enumkeycol: .Q.en[.wdb.hdbdir;?[tablename;enlist (<;`time;nextp);0b;()]];
```

can possibly just use `.wdb.hdbdir` directly instead of creating a local copy of that variable?